spec/shipcloud/address_spec.rb

```diff
@@ -59,7 +63,7 @@
   describe '.all' do
     it 'makes a new Get request using the correct API endpoint' do
       expect(Shipcloud).to receive(:request).
-        with(:get, 'addresses', {}).and_return([])
+        with(:get, 'addresses', {}, api_key: nil).and_return([])
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.